src/utilities/clipboard.ts

```diff
@@ -0,0 +1,7 @@
+export function setClipboardContent(content: string) {
```

Comment:
Can we rename these to be more explicit that it's using sessionStorage to disambiguate from normal clipboard?